frontend/src/features/seeds/api/findAllSeeds.ts

```diff
@@ -1,10 +1,21 @@
-import { SeedDto } from '@/bindings/definitions';
+import { Page, SeedDto } from '@/bindings/definitions';
 import { baseApiUrl } from '@/config';
 import axios from 'axios';
 
-export const findAllSeeds = async (): Promise<SeedDto[]> => {
+export const findAllSeeds = async (page: number, name?: string): Promise<Page<SeedDto>> => {
   try {
-    const response = await axios.get<SeedDto[]>(`${baseApiUrl}/api/seeds`);
+    const pageString: string = page != undefined ? page.toString() : '1';
+
+    const searchParams = new URLSearchParams();
+    searchParams.append('per_page', '10');
```

Comment:
Maybe more per default?

```suggestion
    searchParams.append('per_page', '30');
```

Reply:
https://github.com/ElektraInitiative/PermaplanT/pull/276/commits/0de81c2e625fe70de1edd7a1bae5a97d7555dd5d